industries/services/kis_index_service.py

```
@@ -0,0 +1,350 @@
+
+        try:
+            # 4. 실제 API 호출
+            path = "/oauth2/tokenP"
+            url = f"{self.base_url}{path}"
+            data = {
+                "grant_type": "client_credentials",
+                "appkey": self.app_key,
+                "appsecret": self.app_secret,
+            }
+
+            res = requests.post(url, data=json.dumps(data))
```

Ruff (0.14.13)

Probable use of `requests` call without timeout

(S113)